tcms/telemetry/templates/telemetry/testing/execution-trends.html

```diff
@@ -76,6 +76,21 @@
             </div>
         </form>
 
+
+        <div class="passing-rate-summary">
+            <div class="progress-description" style="display: flex; justify-content: space-between; max-width:100%">
+                <div>
+                    <span class="pficon pficon-ok"></span><strong>Passed</strong>
+                </div>
+                <div>
+                    <span class="pficon pficon-error-circle-o"></span><strong>Not passed</strong>
+                </div>
```

Comment:
for this progress bar is it possible to add 3 sections, stacked together: Positive, Neutral & Negative?